CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 2.3.0 - 2025-03-18
+
+### Changed
```

Comment:
@PLangowski needs to be fixed in `git-cliff`, missing `Added` and `Fixed` sections